### PR TITLE
delete e2etest exe job

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1122,10 +1122,3 @@ workflows:
           report: true
           requires:
             - Build app - Linux (stage)
-      # e2e desktop tests on exe build
-      - e2e-exe:
```
Comment (zalenskiSofteq) [inline .circleci/config.yml:1126]: I think better comment this test
Someday it will return

File: .circleci/config.yml [hunk 2]
```diff
@@ -1122,10 +1122,3 @@ workflows:
...
-          name: E2ETest (exe) - Nightly
-          parallelism: 4
-          report: true
-          requires:
-            - Build app - Windows (stage)
```
Comment (AlenaNaboka) [inline .circleci/config.yml:1131]: Can you  please just comment exe job? We are going to add windows tests when devops set environment for them. 
Like here: https://github.com/RedisInsight/RedisInsight/pull/927/files

Comment (vlad-dargel) [reply .circleci/config.yml:1131]: sure

File: .circleci/config.yml (hunk 1 above)
Comment (vlad-dargel) [reply .circleci/config.yml:1126]: agree. thanks!

**Review by zalenskiSofteq — APPROVED** [review]

**Pull request merged**